examples/Bremsstrahlung/include/simulation_defines/param/gridConfig.param

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright 2013-2017 Rene Widera, Richard Pausch, Benjamin Worpitz
+ *
+ * This file is part of PIConGPU.
+ *
+ * PIConGPU is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PIConGPU is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PIConGPU.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+
+
+#pragma once
+
+namespace picongpu
+{
+
+    namespace SI
+    {
+        /** equals X
+         *  unit: meter */
+        constexpr float_64 CELL_WIDTH_SI = 8e-6 / 2048;
+        /** equals Y - the propagation direction
+         *  unit: meter */
+        constexpr float_64 CELL_HEIGHT_SI = CELL_WIDTH_SI;
+        /** equals Z
+         *  unit: meter */
+        constexpr float_64 CELL_DEPTH_SI = CELL_WIDTH_SI;
+        /** Duration of one timestep
+         *  unit: seconds */
+        constexpr float_64 DELTA_T_SI = CELL_WIDTH_SI / 3e8 / 1.4142135623730951 / 1.01;
+    } //namespace SI
+
+    //! Defines the size of the absorbing zone (in cells)
+    const uint32_t ABSORBER_CELLS[3][2] = {
+        {32, 32},  /*x direction [negative,positive]*/
+        {32, 32},  /*y direction [negative,positive]*/
+        {32, 32}   /*z direction [negative,positive]*/
+    }; //unit: number of cells
+
+    //! Define the strength of the absorber for any direction
+    const float_X ABSORBER_STRENGTH[3][2] = {
+        {1.0e-3, 1.0e-3}, /*x direction [negative,positive]*/
+        {1.0e-3, 1.0e-3}, /*y direction [negative,positive]*/
+        {1.0e-3, 1.0e-3}  /*z direction [negative,positive]*/
+    }; //unit: none
+
+    constexpr uint32_t ABSORBER_FADE_IN_STEPS = 16;
+
+    /** When to move the co-moving window.
+     *  An initial pseudo particle, flying with the speed of light,
+     *  is fired at the begin of the simulation.
+     *  When it reaches slide_point % of the absolute(*) simulation area,
+     *  the co-moving window starts to move with the speed of light.
+     *
+     *  (*) Note: beware, that there is one "hidden" row of gpus at the y-front,
+     *            when you use the co-moving window
+     *  0.75 means only 75% of simulation area is used for real simulation
+     */
+    constexpr float_64 slide_point = 0.90;
+
+}
+
```

Comment:
I removed this very empty line but this stupid github thinks I removed the two *following* empty lines thus it doesn't delete this remark.